cdw/mans/README.md

```diff
@@ -0,0 +1,29 @@
+## CDW Managed Services Kyverno Policies
+These Kyverno policies are targeted for deployment to CDW Managed Services
+Kubernetes clusters.  The intent is to use GitOps with
+[kustomize](https://kustomize.io/) to install.
+
+### How things are organized
+Kustomize provides a solution for customizing Kubernetes resource configuration
+free from templates and DSLs. It lets you customize raw, template-free YAML files
+for multiple purposes, leaving the original YAML untouched and usable as is.
+
+With this in mind, the simplest approach to understanding "how things work" is
+to treat `kustomization.yaml` files as bread crumbs. That is to say, where you
+find a kustomization.yaml file, it will take you to your next
+kustomization.yaml...which in turn will take you to another
+kustomization.yaml...etc., etc.
```

Comment:
Backtick `kustomization.yaml` for consistency.

Reply:
done